x-pack/legacy/plugins/uptime/server/rest_api/index_pattern/get_index_pattern.ts

```diff
@@ -14,11 +14,11 @@ export const createGetIndexPatternRoute: UMRestApiRouteFactory = (libs: UMServer
   options: {
     tags: ['access:uptime'],
   },
-  handler: async ({ savedObjectsClient: client }, _context, _request, response): Promise<any> => {
+  handler: async ({ callES }, _context, _request, response): Promise<any> => {
     try {
       return response.ok({
         body: {
-          ...(await libs.savedObjects.getUptimeIndexPattern(client, undefined)),
+          ...(await libs.savedObjects.getUptimeIndexPattern({ callES })),
```

Comment:
`getUptimeIndexPattern` is expecting just the `callES` function causing an exception here. This can be fixed by replacing `getUptimeIndexPattern({ callES })` with `getUptimeIndexPattern(callES)`

Reply:
Sorry about this, must have messed up while fixing typings here.